tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
@keradus I think I agree with your last comment except for this point:

```
{ '@Bar': false, } : -@Bar => { 'AA' => true, NOOP' => false, } // as direct revert
```

IMO, if your configuration only includes a disabled ruleset with no further rules, the resolved result should be an empty list. In your exemple, this enable `AA` rule.